lib/ui/page/home/page/my_profile/view.dart

```diff
@@ -111,15 +111,17 @@ class MyProfileView extends StatelessWidget {
           child: Scaffold(
             appBar: CustomAppBar(title: _bar(c, context)),
             body: Builder(builder: (context) {
-              final Widget child = ScrollablePositionedList.builder(
-                key: const Key('MyProfileScrollable'),
-                initialScrollIndex: c.listInitIndex,
-                scrollController: c.scrollController,
-                itemScrollController: c.itemScrollController,
-                itemPositionsListener: c.positionsListener,
-                itemCount: ProfileTab.values.length,
-                physics: const ClampingScrollPhysics(),
-                itemBuilder: (context, i) => _block(context, c, i),
+              final Widget child = ListViewObserver(
+                controller: c.observerController,
+                onObserve: c.onObserve,
+                triggerOnObserveType: ObserverTriggerOnObserveType.directly,
+                child: ListView.builder(
```

Comment:
This isn't a part of the issue, so this can be ignored and fixed sometime later, but if possible the scroll position jumping up and down should be accounted as well.

https://github.com/user-attachments/assets/97d36eb5-6761-4e2c-b1ea-fbc24556e2ed

Reply:
Yeah, i have noticed it, but i thought that's a problem for a separate task